networking/gcp/vpc/variables.tf

```diff
@@ -0,0 +1,61 @@
+variable "project" {
+  description = "Project name"
+  type        = string
+}
+
+variable "region" {
+  description = "Region where to deploy the different subnets"
```

Comment:
Do we commit to bind a region to a VPC? In GCP, a VPC is global. Only subnets are regional.